miden-tx/src/host/mod.rs

```diff
@@ -118,7 +118,7 @@ impl<A: AdviceProvider, T: TransactionAuthenticator> TransactionHost<A, T> {
         process: &S,
     ) -> Result<(), TransactionKernelError> {
         let stack = process.get_stack_state();
-        // # => [aux, note_type, sender_acct_id, tag, note_ptr, RECIPIENT, note_idx]
+        // # => [aux, encoded_type_and_ex_hint, sender_acct_id, tag, note_ptr, RECIPIENT, note_idx]
```

Comment:
I think the first word is equivalent to `NOTE_METADATA`. If so, I'd write it like that.

Reply:
I think this change may not have gotten committed.